org.audacious_media_player.Audacious.yaml

```diff
@@ -0,0 +1,165 @@
+# WIP
+id: org.audacious_media_player.Audacious
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: audacious
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-music
+  - --own-name=org.atheme.audacious
+  - --own-name=org.mpris.MediaPlayer2.audacious
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
```

Comment:
Should be dropped if you don't specify `--socket=wayland`.

Reply:
I tested in VM and seems like it works on Wayland. So reverting back for now. I'll ask one user who uses it daily to check thoroughly...